pages/bib/[id]/index.tsx

```diff
@@ -55,17 +57,23 @@ export default function BibPage({
   annotatedMarc,
   isAuthenticated,
   itemPage = 1,
+  viewAllItems = false,
 }: BibPropsType) {
-  const { pathname, push, query } = useRouter()
+  const { push, query } = useRouter()
   const metadataTitle = `Item Details | ${SITE_NAME}`
   const bib = new Bib(discoveryBibResult)
   const displayLegacyCatalogLink = isNyplBibID(bib.id)
 
   const [itemsLoading, setItemsLoading] = useState(false)
   const [itemFetchError, setItemFetchError] = useState(bib.showItemTableError)
+  const [viewAllEnabled, setViewAllEnabled] = useState(viewAllItems)
```

Comment:
`viewAllEnabled` sounds like a feature flag to me. Can you change this variable name?